ID edits for readme

sishida opened the pull request.

tetchel commented on README.md:
```diff
@@ -3,29 +3,34 @@
+Complete both parts of the Codewind installation.
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button. The first part of Codewind installs.
+3. Go to the **Explorer** view group and open the **Codewind** view. The Codewind workspace is now added to the Explorer menu.
```
The second sentence is not accurate, remove it

tetchel commented on README.md:
```diff
@@ -3,29 +3,34 @@
+<Add gif file here. The gif will begin with a title card and then cover all of the actions in the following section.>
+Complete both parts of the Codewind installation.
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button. The first part of Codewind installs.
```
I think "the first part" is pretty confusing. Users know what clicking install does here; it installs the extension. I think the second sentence is not necessary.

tetchel commented on README.md:
```diff
@@ -3,29 +3,34 @@
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+<Add gif file here. The gif will begin with a title card and then cover all of the actions in the following section.>
+Complete both parts of the Codewind installation.
```
?

sishida replied:
This was a placeholder to indicate where we will add the gif file. When I drafted edits last week, we were still in our Microclimate repo, but I will remove this now that the file is in the open Eclipse repo.

tetchel commented on README.md:
```diff
@@ -3,29 +3,34 @@
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button. The first part of Codewind installs.
+3. Go to the **Explorer** view group and open the **Codewind** view. The Codewind workspace is now added to the Explorer menu.
+4. To complete the second part of the installation, expand the Codewind workspace and click any of the functions. A notification window appears and asks if you would like to install the remaining Codewind component.
```
> A notification window appears and asks if you would like to install the remaining Codewind component.
I don't think we have to say this. The user will see it for themselves.

tetchel replied:
I also prefer 

`To complete the second part of the installation,`
->
`To complete the installation`

sishida replied:
I removed mention of "the second part," but I was under the impression that having two buttons to click for the installation was unusual and that it was good to ensure the user that the pop-up wasn't a result of them doing something wrong.

It's also not too out of the ordinary to write down the result of an action as part of a step. :)

tetchel commented on README.md:
```diff
@@ -3,29 +3,34 @@
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button. The first part of Codewind installs.
+3. Go to the **Explorer** view group and open the **Codewind** view. The Codewind workspace is now added to the Explorer menu.
+4. To complete the second part of the installation, expand the Codewind workspace and click any of the functions. A notification window appears and asks if you would like to install the remaining Codewind component.
+5. Click **Install** to complete the Codewind installation. The file is <size of download>. For more information, see <more info link> (same link as in pop up). When installation completes, Codewind automatically connects to your local server and notifies you that it is on and not running.
```
> When installation completes, Codewind automatically connects to your local server and notifies you that it is on and not running.

This doesn't make sense; it can't connect if it is not running. The behaviour is to start the server and then connect to it.

tetchel commented on README.md:
```diff
@@ -3,29 +3,34 @@
 
-If you want to host or build the extension yourself, see [Contributing](#contributing).
+## Using Codewind for VS Code
+After the installation completes, you can use the following features in the Codewind workspace in the Explorer menu.
```
I think this is kind of misleading; this is a very small subset of our available features. I think we should just have the Features list below.

sishida replied:
My concern here was that we should explain the buttons around the Codewind workspace as it appears in the Explorer menu...but now that you mention it, the hover help would probably provide sufficient explanation for these buttons. :)

tetchel commented on README.md:
```diff
@@ -3,29 +3,34 @@
-## Getting started
+## Installing Codewind
+Prerequisites
+- Install [VS Code Version 1.27 or later](https://code.visualstudio.com/download).
```
while we're at it, we can remove the version from this. 1.27 is from last October; this is old enough that any user will be off it by now. let's just say Install [VS Code](link)

jopit commented:
Hi @sishida you need to sign the Eclipse Contributor Agreement before the changes can be merged.

If you log in to eclipse at https://www.eclipse.org/ and then view you account, you'll see a tab at the top labeled 'Eclipse Contributor Agreement' which will take you to a form to fill out.

jopit commented:
You'll also need to amend you commit to include a Signed-off-by statement, see here: https://www.eclipse.org/projects/handbook/#resources-commit

The Signed-off-by statement can automatically be added by adding the `-s` parameter tot the `git commit` command.

tetchel commented:
I realized while reviewing this there are some broken links due to my own find-and-replace failure.

Any link to:
https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools
should instead be:
https://marketplace.visualstudio.com/items?itemName=IBM.codewind

sishida commented:
Have done:
- Thanks for the feedback! I have addressed the comments in the latest commit. :)
- I have created an account and signed the Eclipse Contributor Agreement.

To do:
- I still need to include a Signed-off-by statement. I created this pull request with the web browser, so I will plan to explore the command line more tomorrow.
- We will need to add the gif file for the "Installing Codewind" section. I think Guy can help with the gif.

tetchel commented on README.md:
```diff
@@ -3,29 +3,28 @@
 
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+Complete the Codewind installation.
```
is this a header? should we make it a proper header, or follow it with a colon to make it clear it's introducing these steps?

tetchel commented on README.md:
```diff
@@ -3,29 +3,28 @@
+Complete the Codewind installation.
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button.
+3. Go to the **Explorer** view group and open the **Codewind** view. Click the **open workspace** button to open the Codewind workspace.
```
The open workspace button won't show up until after codewind is installed and started

tetchel commented on README.md:
```diff
@@ -3,29 +3,28 @@
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+Complete the Codewind installation.
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
```
I think we can combine this step and the next one into one step - the big green install button is pretty intuitive

Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).

tetchel commented on README.md:
```diff
@@ -3,29 +3,28 @@
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button.
+3. Go to the **Explorer** view group and open the **Codewind** view. Click the **open workspace** button to open the Codewind workspace.
+4. To complete the installation, expand the Codewind workspace and click any of the functions. A notification window appears and asks if you would like to install the remaining Codewind component.
```
>  To complete the installation, expand the Codewind workspace and click any of the functions. A notification window appears and asks if you would like to install the remaining Codewind component.

->
To complete the installation, open the Codewind view in the Explorer view group.

tetchel commented on README.md:
```diff
@@ -35,10 +34,10 @@ If you want to host or build the extension yourself, see [Contributing](#contrib
+Submit issues and contributions:
+- [Submitting issues](https://github.com/eclipse/codewind/issues)
+- [Contributing](CONTRIBUTING.md)
+- See [development builds](https://public.dhe.ibm.com/ibmdl/export/pub/software/microclimate/vscode-tools/nightly/).
```
let's just remove this link for now because it's out of date

tetchel commented on README.md:
```diff
@@ -35,10 +34,10 @@ If you want to host or build the extension yourself, see [Contributing](#contrib
+- [Submitting issues](https://github.com/eclipse/codewind/issues)
+- [Contributing](CONTRIBUTING.md)
+- See [development builds](https://public.dhe.ibm.com/ibmdl/export/pub/software/microclimate/vscode-tools/nightly/).
+- Follow the [Install from a VSIX](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) instructions to install a `.vsix`.
```
this too because it relies on the previous line

tetchel commented on README.md:
```diff
@@ -3,29 +3,28 @@
-
-## Features
-- Create new projects from application templates, or import existing docker-ready projects.
+You can use the following features:</br>
```
I know we're trying to make the user the actor, but this reads really bizarrely to me. I liked just "Features" better.

tetchel commented:
added more comments, previous conversation is out of date but it looks like i don't have permissions to resolve those

tetchel commented:
this can be closed, superceded by https://github.com/eclipse/codewind-vscode/pull/9/files

jopit commented:
Closing as per Tim's comment.
